WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersSettingsFlow.swift

```diff
@@ -0,0 +1,215 @@
+        addChild(host)
+        view.addSubview(host.view)
+        view.pinSubviewToAllEdges(host.view)
+        host.didMove(toParent: self)
+    }
+
+    func makeCoordinator() -> BlogRemindersCoordinator {
+        let coordinator = BlogRemindersCoordinator()
+        coordinator.presenter = self
+        return coordinator
+    }
+
+    var collapsedHeight: DrawerHeight {
+        .intrinsicHeight
+    }
+
+    var allowsDragToDismiss: Bool {
+        true
+    }
+}
+
+struct BloggingRemindersSettingsIntroView: View {
+    @EnvironmentObject var coordinator: BlogRemindersCoordinator
+
+    var body: some View {
+        VStack(spacing: Metrics.introStackSpacing) {
+            Image(systemName: Images.starImageName)
+                .resizable().frame(width: Metrics.topImageSize.width, height: Metrics.topImageSize.height, alignment: .center)
+            Text(TextContent.introTitle)
+                .font(TextContent.introTitleFont)
+            Text(TextContent.introDescription)
+            NavigationLink(destination: BloggingRemindersSettingsView().environmentObject(coordinator)) {
+                Text(TextContent.getStartedButtonTitle)
+                    .primaryButtonStyle()
+            }
+            Spacer()
+        }
+        .padding()
+        .frame(maxWidth: .infinity, maxHeight: .infinity)
+        .navigationBarHidden(true)
+    }
+}
+
+struct BloggingRemindersSettingsView: View {
+    @EnvironmentObject var coordinator: BlogRemindersCoordinator
+
+    // TODO: This will be replaced by some observable object or similar that
+    // coordinates with our reminders store. @frosty
+    @SwiftUI.State private var toggles: [Bool] = [false, false, false, false, false, false, false]
+
+    let days: [String] = {
+        var calendar = Calendar.current
+        calendar.locale = Locale.autoupdatingCurrent
+        let firstWeekday = calendar.firstWeekday
+        var symbols = calendar.shortWeekdaySymbols
+
+        let localizedWeekdays: [String] = Array(symbols[firstWeekday - 1 ..< Calendar.current.shortWeekdaySymbols.count] + symbols[0 ..< firstWeekday - 1])
+
+        return localizedWeekdays
+    }()
+
+    var body: some View {
+        VStack(spacing: Metrics.settingsStackSpacing) {
+            Spacer()
+            Image(systemName: Images.calendarImageName)
+                .resizable()
+                .frame(width: Metrics.topImageSize.width, height: Metrics.topImageSize.height, alignment: .center)
+            VStack(spacing: Metrics.settingsInnerStackSpacing) {
+                Text(TextContent.settingsPrompt)
+                    .font(TextContent.settingsPromptFont)
+                    .fontWeight(.semibold)
+                    .lineLimit(nil)
+                    .multilineTextAlignment(.center)
+                Text(TextContent.settingsUpdatePrompt)
+                    .foregroundColor(.gray)
+                    .lineLimit(2)
+            }
+            HStack {
```

Comment:
I'd consider making a component (custom view) out of this, since the day picker could eventually come handy for other purposes, and since it's going to help modularize the logic a bit.

That said - if we want to keep the ball rolling for now, I'm good with leaving it as it is in this PR.

Reply:
Yeah, I definitely think pulling it out makes sense – however this is likely to chance considerably in a follow-up PR (the days need to wrap lines and use real data), so I think it'll be fine to do it then. Thanks for the prompt!